packages/tocco-ui/src/EditableValue/EditableValue.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import typeEditorFactory from './typeEditorFactory'
+
+import './styles.scss'
+
+/**
+ *  To edit values of given type.
+ */
+const EditableValue = props => {
+  return (
+    <span className="tocco-editable-value">
+      {
+        typeEditorFactory(props.type, props.value, props.onChange, props.options)
+      }
+    </span>
+  )
+}
+
+EditableValue.propTypes = {
+  /**
+   * Type of value. e.g. 'string'
+   */
+  type: React.PropTypes.oneOf([
+    'multi-select',
+    'string'
+  ]).isRequired,
```

Comment:
This list should be coming from a source like the map in typeEditorFactory. It might get cumbersome to keep this list up to date at all times here.